### PR TITLE
Add demos

### PR DESCRIPTION
**Pull request by donpaul999**
Add user demo and recipe creator demo.

### CODE REVIEW

**Review by bessmarche — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,38 @@
...
+Our Users
+----------
+
+**Casual Users:**
+
+Mainly searching for recipes. They do not have an account. Visit the website sporadically, only when looking for recipe inspiration.
+
+**Recipe Creators:**
+
+Use the site to post recipes. They have an account and use it mainly to search for recipes. In addition they want to modify their own existing recipes or post brand new recipe to the site.
```
Comment (ramaroberto) [inline README.md]: I don't think you need this bit.

File: README.md [hunk 2]
```diff
@@ -1 +1,38 @@
-Welcome to Piece of Cake! :D
+Welcome to Piece of Cake!
+-------------------------
+We are Beatrice and Paul, two Google STEP Interns (Summer 2020) in the Plx Alerts team.
+
+In the second half of our internship we worked on a full stack web development project. 
+
+We designed and implemented "Piece Of Cake" a platform where everyone, regardless of their tech abilities can post and share their favourite recipes. We believe that information about how to cook good and healthy food should be accessible to as wide of an audience as possible that is why we have decided to make searching for/accessing recipes available to all users of the website regardless of whether they have an account or not.
+
+The website consists of the following pages: <br>
```
Comment (ramaroberto) [inline README.md:9]: Are you sure you need the <br> here and below?

File: README.md [hunk 3]
```diff
@@ -1 +1,38 @@
...
+* Home/Search, for the users to search for recipes; <br>
```
Comment (ramaroberto) [inline README.md]: Why are you using semicolons here? This is already an itemization :).

File: README.md [hunk 4]
```diff
@@ -1 +1,38 @@
...
+
+Mainly searching for recipes. They do not have an account. Visit the website sporadically, only when looking for recipe inspiration.
+
+**Recipe Creators:**
+
+Use the site to post recipes. They have an account and use it mainly to search for recipes. In addition they want to modify their own existing recipes or post brand new recipe to the site.
+
+
+
+Demo for casual user
```
Comment (ramaroberto) [inline README.md]: Searching for a recipe demo

File: README.md [hunk 5]
```diff
@@ -1 +1,38 @@
...
+--------------------
+![alt-text](demo/demo_user.gif)
+
+Demo for recipe creator
```
Comment (ramaroberto) [inline README.md]: Creating a recipe demo

**Review by ramaroberto — APPROVED** [review]

**Pull request merged**